src/modules/auth/reducer.js

```diff
@@ -2,6 +2,8 @@ import { types } from './actions'
 
 const initialState = {
 	userinfo: null,
```

Comment:
Another remark here is that 'tenants' for given credentials should be part of `userinfo` call. No dedicated API call should be necessary for obtaining 'current' tenants, roles, etc..

If SeaCat Auth don't provide this now, please communicate with PLUM team.

Reply:
Sure, I can get the tenants for given credentials from `userInfo`, but I dont see the point, because authorization dont need the user tenants. We might need them if we will decide, that in the TenantDropdown will be displayed all the tenants assigned to the user, not only the authorized tenants. But this is what I mentioned in `1]` question on you on my latest comment.

Anyway, in this case, the reducer pass the payload of authorized tenants, which has been allowed to use within the application (this is substracted from the `rbac` endpoint, but this endpoint require the tenants of the application, not the user's). This payload is passed as a props to the TenantDropdown and then displayed. So only tenants for which user has authorization are displayed.

So, there is no dedicated API call for obtaining `current` tenants, roles, etc... But there is an API call for obtaining all the application's tenants, because they are needed for authorization procedure, which I have mentioned above.